install-guide.py

```diff
@@ -160,8 +160,9 @@ def Check_Missing_Dep():
                     continue
 
 def Check_dlib():
-    if not 'dlib' in Missing_Packages or not ENABLE_CUDA:
+    if 'dlib' in Missing_Packages:
         Required_Packages.insert(0, 'cmake')

```

Comment:
Required_Packages.insert(0, XXX) should not happen because Required_Packages[0] is supposed to be always tensorflow(-gpu). It's used in Update_TF_Dep()

Reply:
It has to be installed prior to dlib, and update_tf_dep has already happened. It could be inserted into position 1 though.

Reply:
you are right. I am integrating now.